holidays/countries/falkland_islands.py

```diff
@@ -0,0 +1,139 @@
+    """
+
+    country = "FK"
+    default_language = "en_GB"
+    supported_languages = ("en_GB", "en_US")
+    supported_categories = (GOVERNMENT, PUBLIC)
+    # %s observed.
+    observed_label = tr("%s (observed)")
+
+    start_year = 1983
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        StaticHolidays.__init__(self, cls=FalklandIslandsStaticHolidays)
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day(tr("New Year's Day")))
+
+        # Good Friday.
+        self._add_good_friday(tr("Good Friday"))
+
+        if self._year <= 2022:
+            # Queen's Birthday.
+            self._add_observed(self._add_holiday_apr_21(tr("Queen's Birthday")))
+
+        # Liberation Day.
+        self._add_observed(self._add_holiday_jun_14(tr("Liberation Day")))
+
+        if self._year <= 2001:
+            # Falkland Day.
+            self._add_observed(self._add_holiday_aug_14(tr("Falkland Day")))
+        else:
+            # Peat Cutting Day.
+            self._add_holiday_1st_mon_of_oct(tr("Peat Cutting Day"))
+
+        # King's Birthday.
+        self._add_observed(self._add_holiday_nov_14(tr("King's Birthday")))
+
+        dec_25 = (DEC, 25)
+        christmas_day_rule = None
+        christmas_day_two_rule = SAT_SUN_TO_NEXT_MON_TUE
+        christmas_day_three_rule = SAT_SUN_TO_NEXT_MON_TUE
+        if self._is_saturday(dec_25):
+            christmas_day_rule = SAT_SUN_TO_NEXT_TUE_WED
+            christmas_day_two_rule = SAT_SUN_TO_NEXT_TUE_WED
+        elif self._is_sunday(dec_25):
+            christmas_day_rule = SUN_TO_NEXT_WED
+
+        # Christmas Day.
+        self._add_observed(self._add_christmas_day(tr("Christmas Day")), rule=christmas_day_rule)
+
+        self._add_observed(
+            # Boxing Day.
+            self._add_christmas_day_two(tr("Boxing Day")),
+            rule=christmas_day_two_rule,
+        )
+
+        self._add_observed(
+            # Christmas Holiday.
+            self._add_christmas_day_three(tr("Christmas Holiday")),
+            rule=christmas_day_three_rule,
+        )
+
+    def _populate_government_holidays(self):
+        dec_30 = (DEC, 30)
+        # Government Holiday.
+        name = tr("Government Holiday")
+        govt_holiday_one_rule = SAT_SUN_TO_PREV_THU_FRI
+        govt_holiday_two_rule = SAT_SUN_TO_PREV_THU_FRI
+        if self._is_friday(dec_30):
+            govt_holiday_two_rule = SAT_TO_PREV_THU
+
+        self._add_observed(self._add_holiday_dec_30(name), rule=govt_holiday_one_rule)
+        self._add_observed(self._add_holiday_dec_31(name), rule=govt_holiday_two_rule)
```

Comment:
It looks like the "last two working days of the year" rule. So... a bit of working day magic:
```suggestion
        # Government Holiday.
        name = tr("Government Holiday")
        last_workday = self._add_holiday(
            name, self._get_next_workday(self._next_year_new_years_day, -1)
        )
        self._add_holiday(name, self._get_next_workday(last_workday, -1))
```